app/views/games/index.html.erb

```diff
@@ -1,5 +1,58 @@
-<ul>
-  <% @games.each do |game| %>
-    <li><%= game.title %></li>
-  <% end %>
-</ul>
+<div class="container-fluid">
+  <div class="row">
+    <div class="col">
+      <% @games.each do |game| %>
+        <div class="game-card ">
+          <div class="picture">
+            <%= instance_image_tag game.photo, height: 200, width: 315, crop: :fill %>
+          </div>
+          <div class="game-content">
+            <h4><%= game.title %></h4>
+            <% if game.developer? %>
+              <p><span>Développeur </span><%= game.developer %></p>
+            <% end %>
+            <% if game.editor? %>
+              <p><span>Editeur </span><%= game.editor %></p>
+            <% end %>
+            <% if game.franchise? %>
+            <p><span>Franchise </span><%= game.franchise %></p>
+            <% end %>
+            <% if game.genre? %>
+              <p><span>Genre </span><%= game.genre %></p>
+            <% end %>
+            <% if game.year? %>
+              <p class="game-published"><span>date de parution </span><%= game.year%></p>
+            <% end %>
+          </div>
+        </div>
+        <div class="game-card-mobile">
+          <div class="picture-mobile">
+            <%= instance_image_tag game.photo, effect:"gradient_fade,y_-0.5,b_rgb:050505", crop: :fill%>
```

Comment:
what `effect` does?

Reply:
As you can see in the previous comment, I added a gradient filter over the image, to see better the white title. That's how Cloudinary handles special effects on images.